src/Npgsql.NetTopologySuite/NetTopologySuiteHandlerFactory.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using NetTopologySuite.Geometries;
+using NetTopologySuite.IO;
+using Npgsql.PostgresTypes;
+using Npgsql.TypeHandling;
+
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+
+namespace Npgsql.NetTopologySuite
+{
+    public class NetTopologySuiteHandlerFactory : NpgsqlTypeHandlerFactory<Geometry>
```

Comment:
I don't really see a reason for this to have its own file - we usually just keep the factory in the same file as its handler. But no strong feels if you really prefer it this way.

Reply:
It's a rule in .NET Core repos by the way, and I think it's good enough to follow. The reason is that the code base looks cleaner and is easier to maintain for new contributors.

Reply:
I don't really agree, an explosion of files for little classes makes the codebase harder to navigate and grok.